rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -287,16 +287,14 @@ rocalSequenceReader(
         bool shuffle,
         bool loop,
         unsigned step,
-        unsigned stride)
-{
-    Image* output = nullptr;
+        unsigned stride) {
+    rocalTensor* output = nullptr;
```

Comment:
please move this after the context validity check

Reply:
We are checking if the context is set and we throw the error and return the rocalTensor
Hence the output rocalTensor is initialized to nullptr before the validity check